contracts/IRrpBeaconServer.sol

```diff
@@ -1,2 +1,2 @@
 // SPDX-License-Identifier: MIT
-import "@api3/airnode-protocol/contracts/rrp/requesters/interfaces/IRrpBeaconServer.sol";
+import '@api3/airnode-protocol/contracts/rrp/requesters/interfaces/IRrpBeaconServer.sol';
```

Comment:
I don't think this will compile

Reply:
(used [here](https://github.com/api3dao/beacon-reader-example/blob/main/contracts/BeaconReaderExample.sol#L4))

Reply:
This was the result of prettier... I'll try quick and revert.

Reply:
Not the error I was expecting:
If you run `hardhat compile` in services it complains: (it needs `solidity: "0.8.9"` added to `hardhat.config.ts` - should I fix this?)
```
Solidity compiler is not configured. Version 0.7.3 will be used by default. Add a 'solidity' entry to your configuration to suppress this warning.

Learn more about compiler configuration at https://hardhat.org/config

Error HH606: The project cannot be compiled, see reasons below.

The Solidity version pragma statement in these files doesn't match any of the configured compilers in your config. Change the pragma or configure additional compiler versions in your hardhat config.

  * @api3/airnode-protocol/contracts/rrp/requesters/interfaces/IRrpBeaconServer.sol (0.8.9)

These files and its dependencies cannot be compiled with your config. This can happen because they have incompatible Solidity pragmas, or don't match any of your configured Solidity compilers.

  * contracts/IRrpBeaconServer.sol

```

Reply:
If I use the CLI to create an example reader app and then run `hardhat compile` I get:
```
Error HH14: Your Hardhat project uses typescript, but it's not installed.
```
Should I fix this (or perhaps I'm doing something wrong)

Reply:
No, we don't need `@api3/services` to compile this contract. We're just importing the contract and exposing it to the user of `@api3/services` so that they don't need to worry about what `@api3/airnode-protocol` version they should use.

Reply:
I guess the root `package.json` shouldn't have a hardhat dependency anyway. We can leave it as is and revisit after the hackathon.

Reply:
All good.
It compiles but the Solidity style guide indicates `"` are preferred, also `airnode-examples` contracts use `"`.
I'll revert the change and alter `prettier` so it uses `"` for .sol based on `airnode`.

Reply:
Done, please confirm :smile: